logstash-output-opensearch.gemspec

```diff
@@ -1,3 +1,5 @@
+signing_key_path = File.expand_path("gem-private_key.pem")
```

Comment:
you don't need to expand if it is just name.

Reply:
Can you also test locally by following the steps mentioned here: https://guides.rubygems.org/security/

Reply:
Updated